evm_arithmetization/src/cpu/kernel/constants/mod.rs

```diff
@@ -320,8 +331,38 @@ pub mod cancun_constants {
         balance: U256::zero(),
         // Storage root for this account at genesis.
         storage_root: H256(hex!(
-            "f58e5f1eae7ce386de44266ff0286a88dafe7e4269c1ffa97f79dbbcf4f59e5c"
+            // "f58e5f1eae7ce386de44266ff0286a88dafe7e4269c1ffa97f79dbbcf4f59e5c"
```

Comment:
Not sure what this root is and why it's not the empty trie root (`0x56e81f...`)? @Nashtare

Reply:
Actually this is a leftover from an initial version of the impl (can't recall why I put this value tbh) but we're not using this constant anymore (it's replaced by the `initial_state_and_storage_tries_with_beacon_roots()` utility method, which does specify an empty storage root) so could simply be removed?